app/views/accounts/_bio.html.haml

```diff
@@ -1,4 +1,17 @@
 .public-account-bio
+  - if account.account_identity_proofs
+    .account__header__fields
+      - account.account_identity_proofs.each do |proof|
```

Comment:
i think this should only be showing proofs that are live.

Reply:
there's already a scope on account_identity_proofs, so if you change it to...
```
account.account_identity_proofs.active.each
```
it should work. might be worth pulling the logic up to the controller, but i can do that on another pass.